srcpkgs/pkgconf/template

```diff
@@ -13,20 +12,10 @@ homepage="https://git.dereferenced.org/pkgconf/pkgconf"
 changelog="https://git.dereferenced.org/pkgconf/pkgconf/raw/branch/master/NEWS"
 distfiles="https://distfiles.dereferenced.org/pkgconf/${pkgname}-${version}.tar.xz"
 checksum=d3468308553c94389dadfd10c4d1067269052b5364276a9d24a643c88485f715
+configure_args="--disable-shared --disable-static"
 
 post_install() {
 	vlicense COPYING
-}
-
-pkgconf-devel_package() {
-	depends="pkgconf-${version}_${revision}"
-	short_desc+=" - development files"
-	pkg_install() {
-		vmove usr/include
-		vmove usr/lib/pkgconfig
-		vmove usr/share/aclocal
-		vmove usr/share/man/man7
-		vmove "usr/lib/*.so"
-		vmove "usr/lib/*.a"
-	}
+	rm -r $DESTDIR/usr/include
+	ln -sr $DESTDIR/usr/bin/pkgconf $DESTDIR/usr/bin/pkg-config
```

Comment:
should be 

ln -s /usr/bin/pkgconf $DESTDIR/usr/bin/pkg-config

also problably nice to have

ln -s /usr/share/man/man1/pkgconf.1 $DESTDIR/usr/share/man/man1/pkg-config.1

Reply:
1. `-r` takes care about making the link relative. This safes some bytes and makes it more robust if rootdir != /
2. okay.

Reply:
1. noted
2. thanks